policies/lib/Cargo.toml

```
@@ -0,0 +1,18 @@
+[package]
+name = "lib"
+version = "0.1.0"
+edition = "2021"
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+serde = { version = "1", features = ["derive"] }
+rudderc = { path = "../rudderc", features = ["embedded-lib"] }
```

```suggestion
rudderc = { path = "../rudderc" }
```

I don't think we want this feature here.